src/java.desktop/macosx/classes/apple/laf/JRSUIConstants.java

```diff
@@ -112,7 +112,7 @@ public boolean equals(final Object obj) {
 
         public int hashCode() {
             final long bits = Double.doubleToLongBits(doubleValue);
-            return (int)(bits ^ (bits >>> 32));
+            return Long.hashCode(bits);
```

Comment:
This one and the `DoubleDV` one should actually delegate to `Double.hashCode`, which delegates to `Long.hashCode` after this change.

Reply:
Good point, done!